qdrant-landing/content/articles/qdrant-1.7.x.md

```diff
@@ -0,0 +1,112 @@
+Those sparse vectors are theoretically highly dimensional, definitely way higher than the dense vectors used in semantic search. However, since the majority of dimensions are usually zeros, we store them differently and just keep the non-zero dimensions. 
+
+Until now, Qdrant has not been able to handle sparse vectors natively. Some were trying to convert them to dense vectors, but that was not the best solution or a suggested way. We even wrote a piece with [our thoughts on building a hybrid search](/articles/hybrid-search/), and we encouraged you to use a different tool for keyword lookup. 
+
+Things have changed since then, as so many of you wanted a single tool for sparse and dense vectors. That’s why we decided to implement native support for sparse vectors! Please wait closely for our next article, describing how to use this feature in practice. For now, you can check out the [sparse vectors documentation](/documentation/concepts/search/#sparse-vectors)!
+
+TODO: Add a link to the sparse vectors documentation.
+
+### Discovery API
+
+The newly introduced [Discovery API](/documentation/concepts/explore/#discovery-api) aims to enable even more scenarios for using vectors. Its interface is similar to the [Recommendation API](/documentation/concepts/explore/#recommendation-api), but it effectively constraints the search space.
+There is a concept of `context`, which is a set of positive-negative pairs. Each pair divides the space into positive and negative zones. In that mode, the search operation prefers points based on how many positive zones they belong to (or how much they avoid negative zones).
+
+The Discovery API can be used in two ways - either with or without the target point. The first case is called a **discovery search**, while the second is called a **context search**.
+
+#### Discovery search
+
+*Discovery search* is an operation that uses a target point to find the most relevant points in the collection, while performing the search in the preferred areas only. That is basically a search operation with more control over the search space.
+
+![Discovery search visualization](/articles_data/qdrant-1.7.x/discovery-search.png)
+
+Please refer to the [Discovery API documentation on discovery search](/documentation/concepts/explore/#discovery-search) for more details and the internal mechanics of the operation.
+
+#### Context search
+
+The mode of *context search* is similar to the discovery search, but it does not use a target point. Instead, the `context` is used to navigate the HNSW graph towards preferred zones. It is expected that the results in that mode will be diverse, and not centered around one point.
+*Context search* might be an answer for those looking for some kind of exploration of the vector space. 
```

Comment:
Suggestion to rewrite as : 
*Content Search* could serve as a solution for individuals seeking a more exploratory approach to navigate the vector space.

Reply:
thank you !!  :)